docs/how_to/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _how_to:
+
+######
+How to
+######
+
+Create quantum circuits
```

Comment:
I'm not sure how to explain this but it doesn't sound as fluent to say "create" + `<plural items>`. It sounds better to say "creating" + `<plural items>` or "create a" + `<single item>`
```suggestion
Create a quantum circuit
```